pkg/api/types.go

```diff
@@ -768,6 +768,9 @@ type LiteralTestStep struct {
 	// Cli is the (optional) name of the release from which the `oc` binary
 	// will be injected into this step.
 	Cli string `json:"cli,omitempty"`
+	// ClusterClaimCLI defines if this step should inject the `oc` binary for
+	// the cluster claimed by this step. Requires test to define a cluster claim.
+	ClusterClaimCLI *bool `json:"cluster_claim_cli,omitempty"`
```

Comment:
This sounds really unfortunate to me. This would require step authors to decide whether the step is usable on either claimed clusters or installed clusters, but never both, no?

Could we infer `true` here internally from nonempty `Cli`?

Reply:
Maybe infer that `cli: latest` would allow for CLI from the claim as well? I think we *do* need to make authors choose if they are using some weird custom CLI, since that will be doing something different ....